### PR TITLE
Ep13227 graphics app improvements

### PR DESCRIPTION
**Pull request by seishin4real**
changed angular packages versions to fixed (removed ^)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   "main": "./index.ts",
   "scripts": {
     "package": "ng-packagr -p package.json",
-    "dev-release": "npm run package && cd dist/ && npm publish --tag dev",
+    "dev-release": "npm run package && cd dist/ && npm publish --tag legacy",
```
Comment (JerryDoubleU) [inline package.json]: Is this changed to 'legacy' on purpose?

File: package.json [hunk 2]
```diff
@@ -40,28 +40,28 @@
     "playground": "angular-playground"
   },
   "dependencies": {
-    "@angular/animations": ">=7.0.0",
-    "@angular/cdk": "^7.0.4",
-    "@angular/material": "^7.0.3",
+    "@angular/animations": "=7.0.4",
```
Comment (JerryDoubleU) [inline package.json]: Is the version correct "=7.0.4"?

File: package.json (hunk 1 above)
Comment (seishin4real) [reply package.json]: no, my bad

File: package.json (hunk 2 above)
Comment (seishin4real) [reply package.json]: yeah, the = can be dropped

**Pull request merged**